R/apply_row_grp_plan.R

```diff
@@ -58,7 +58,12 @@ apply_row_grp_struct <- function(.data, row_grp_struct_list, group, label = NULL
           unlist() %>%
           nchar() %>%
           max(na.rm = TRUE)
-      } else{
+      }
```

Comment:
Could this be simplified by casting to character in the beginning? I don't think the conditionals are needed.

```r
x %>% 
  as.character() %>%
  # str_split always produces a list even it the pattern is not present in the string
  str_split("\\n") %>% 
  unlist() %>%
  ...
```

or

```r
str_split(..., simplify = TRUE)
```